public/static/docs/command-reference/list.md

```diff
@@ -0,0 +1,40 @@
+# list
+
+List file structure of a repo including <abbr>data artifacts</abbr>
+
+## Synopsis
+
+```usage
+usage: dvc list [-h] [-q | -v] [-R] [--outputs-only] [--rev [REV]] url [target]
+
+positional arguments:
+  url              Url to the repo. Can be either local fs path or git+ssh/https url
+  target           Path to directory within the repository to list outputs for.
+
+optional arguments:
+  --outputs-only   Show only dvc outputs.
+  --rev [REV]      Git revision (e.g. branch, tag, SHA)
+```
+
+## Description
+
+The command list files and <abbr>data artifacts</abbr> into pointed repo.
+
+## Options
+
+- `--outputs-only` - Show only <abbr>data artifacts</abbr>.
+
+- `-R`, `--recursive` - Recursively traverse the repo.
```

Comment:
the repo -> the target? actually it's is the path (that's why I don't like the target name for the CLI argument)

Reply:
check the aws s3 ls and ls man pages. traverse is not the right verb here ... print, list, etc are better

Reply:
Agree. If `target` can only be a directory path, why not call it `path` or `dir`? (Also commenting in iterative/dvc.org/pull/967)